src/pages/Home.js

```diff
@@ -19,18 +20,13 @@ export default function Home({ fetchGuests }) {
         <WelcomeMessage>
           {guestArray.length > 0
             ? "Who do you want to cook for today?"
-            : "Nobody here... Fetch some example guests or add new guests!"}
+            : "Nobody here... Login or add new guests!"}
         </WelcomeMessage>
-
-        {guestArray.length === 0 && (
-          <button type="button" onClick={() => fetchGuests()}>
-            Fetch some guests
-          </button>
-        )}
-
+        <LoginSection />
+        <hr />
         <StyledGuestList>
-          {guestArray.map((guest) => {
-            return <GuestCard key={guest._id} personalData={guest} />;
+          {guestArray.map((guest, index) => {
+            return <GuestCard key={index} personalData={guest} />;
```

Comment:
The `index` is not a stable key. If you change the mapping, for instance by deleting an item in the list, the index of all successive elements in that list changes and so does the key for these elements. But the key shouldn't change to no element. It needs to be unique and stable across render cycles. That's why an id is always a preferable choice.

Reply:
My problem is: 
when you create a new guest, there is no guest.id that you could use as a key.
Every guest gets an id when you save the guest to the dataBase for the first time (to prevent one guest having two ids).
So I need another key than id for the newly created guests - I just don't know which? 😅

Reply:
I see. In that case, you can apply a little hack and map your `guestArray` to itself plus a new id field which corresponds to the current index. Sounds weird at first, but you are creating a stable version of ids that way because you assign a concrete index to an id, which never changes again.